src/content/docs/en/guides/partials.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Partials
+description: Learn how to create partial pages for use with dynamic front-end libraries.
+i18nReady: true
+---
+
+import Badge from '~/components/Badge.astro';
+import Since from '~/components/Since.astro'
+
+<Since v="3.4.0" /> <Badge>Experimental</Badge>
+
+Astro supports pages that are rendered as *partials*. With partials you can load part of a page dynamically in the client without changing the URL with the aide of client-side code. When Astro renders a partial page it omits the `<doctype>` declaration and any `<head>` content, such as scoped styles and scripts.
```

Comment:
Astro supports pages that are rendered as partials. With partials you can load part of a page dynamically on the client-side without changing the URL, with the help of client-side code. When Astro renders a partial page, it omits the <doctype> declaration and any <head> content, such as scoped styles and scripts.